src/app/rundown/components/rundown/rundown.component.html

```diff
@@ -1,17 +1,13 @@
 <sofie-rundown-header *ngIf="rundown" class="c-rundown-header" [rundown]="rundown"></sofie-rundown-header>
 
 <div *ngIf="rundown?.isActive" class="test-buttons">
-    <ng-container *ngIf="!rundown?.isActive; else activeRundownButtons"></ng-container>
+    <button mat-raised-button color="primary" style="margin-left: 20px" (click)="takeNext()">Take Next</button>
```

Comment:
We should really begin to encapsulate components like this into our own components.
Just like you have done with the `IconButton`, we should also have a regular `Button` where the material design is encapsulated it.
Same goes for all the other components like `mat-card`, `mat-table` etc.

Not sure this is the PR to do it in, but it's an FYI

Reply:
Very good point, we should make a task for this, or write it into tasks that relates. I really think we should limit how much this PR is doing at the moment, it is part of the sprint goal and we should get this through first. 
I would gladly do this as part of something else, or as a standalone task at some point.